src/files-card-body.jsx

```diff
@@ -205,53 +213,121 @@ export const FilesCardBody = ({
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "ArrowUp":
+                if (e.altKey && !e.shiftKey && !e.ctrlKey) {
+                    goUpOneDir();
+                } else if (hasNoKeydownModifiers(e)) {
+                    setSelected(_selected => {
+                        const firstSelectedName = _selected?.[0]?.name;
+                        const selectedIdx = sortedFiles?.findIndex(file => file.name === firstSelectedName);
+                        const newIdx = Math.max(selectedIdx - boxPerRow, 0);
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "ArrowDown":
+                if (e.altKey && !e.shiftKey && !e.ctrlKey && selected.length === 1) {
+                    onDoubleClickNavigate(selected[0]);
+                } else if (hasNoKeydownModifiers(e)) {
+                    setSelected(_selected => {
+                        const firstSelectedName = _selected?.[0]?.name;
+                        const selectedIdx = sortedFiles?.findIndex(file => file.name === firstSelectedName);
+                        const newIdx = Math.min(selectedIdx + boxPerRow, sortedFiles.length - 1);
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "Enter":
+                if (hasNoKeydownModifiers(e) && selected.length === 1) {
+                    onDoubleClickNavigate(selected[0]);
+                }
+                break;
+
+            case "Delete":
+                if (hasNoKeydownModifiers(e) && selected.length !== 0) {
+                    Dialogs.show(
+                        <ConfirmDeletionDialog
+                          selected={selected} path={currentPath}
+                          setSelected={setSelected}
+                        />
+                    );
+                }
+                break;
+
+            case "F2":
+                if (hasNoKeydownModifiers(e) && selected.length === 1) {
+                    Dialogs.show(
+                        <RenameItemModal
+                          path={path}
+                          selected={selected[0]}
+                        />
+                    );
+                }
+                break;
+
+            case "a":
+                if (e.ctrlKey && !e.shiftKey && !e.altKey) {
+                    e.preventDefault();
+                    setSelected(sortedFiles);
+                }
+                break;
+
+            case "L":
+                if (!e.ctrlKey && !e.altKey) {
+                    dispatchEvent(new Event("manual-change-dir"));
+                }
+                break;
+
+            case "N":
+                if (!e.ctrlKey && !e.altKey) {
+                    Dialogs.show(
+                        <CreateDirectoryModal currentPath={currentPath} />
```

Comment:
These 4 added lines are not executed by any test.

Reply:
ooops, looks like this slipped my mind!